leftwm/src/config/keybind.rs

```diff
@@ -173,6 +173,7 @@ impl Modifier {
     }
 }
 
+#[cfg(feature = "lefthk")]
```

Comment:
may I ask, why we need it here?

Reply:
Because this function is only used in keybinds, and without the feature flag `&Config` is not in scope, so compilation breaks.